src/auditwheel/manylinux.rs

```diff
@@ -3,7 +3,7 @@ use serde::{Deserialize, Serialize};
 use std::fmt;
 use std::str::FromStr;
 
-/// Decides how to handle manylinux compliance
+/// Decides how to handle manylinux and musllinux compliance
 #[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq, Copy)]
 pub enum Manylinux {
```

Comment:
The `--manylinux` command line option also needs some tweaking.

Reply:
Difficult question. I worry that having both `--target` and `--platform` will be confusing to people, esp. since platform gets. Maybe something like `--compatibility`? `--tag` would be another possibility, but "tag" does sound more like metadata (e.g. docker tag) than like libc flavor and version.